config.toml

```diff
@@ -1,14 +1,14 @@
 base_url = "https://limeleaf.io"
 title = "Limeleaf Worker Collective | We Build Software"
-description = "Limeleaf is a software development cooperative that provides Staff-level software engineering and product development for businesses, brands, and startups."
+description = "Limeleaf is a software development cooperative that draws on decades of experience to help teams build software products faster."
 
-generate_feed = true
-feed_filename = "rss.xml"
+generate_feeds = true
+feed_filenames = ["rss.xml"]
```

Comment:
Positive this is the way even though we use `shalzz/zola-deploy-action@v0.17.2` in the GitHub Action?

Reply:
We might need to bump the version here too based on one here https://github.com/shalzz/zola-deploy-action/tags to match the version of Zola that this is supported in.

Reply:
A couple weeks ago, Zola was giving me startup errors on localhost with

 ```Error: Failed to build the site
Error: TOML parse error at line 6, column 1
  |
6 | feed_filename = "rss.xml"```

I changed that to an array, and the error went away. Can't remember where I found the solution, Zola forums, maybe?

Reply:
Weird. What version do you have locally then so we can all make sure we have what you have, especially GH.

Reply:
zola 0.19.1

Reply:
Bump this then too in this PR so it'll build correctly when merged.

https://github.com/limeleaf-collective/limeleaf.io/blob/main/.github/workflows/zola.yml#L11

Reply:
Bump it to `zola-deploy-action@v0.19.1`?

Reply:
Yup. That way the GitHub Action will match your local version so it will know the new bracket syntax when it goes to build/deploy the site. Otherwise the current version will prob throw a parse error with the new bracket syntax.